homeassistant/components/input_select.py

```diff
@@ -139,3 +173,19 @@ def select_option(self, option):
             return
         self._current_option = option
         self.update_ha_state()
+
+    def select_next(self):
```

Comment:
I could change to that, but I think it would make the interface a bit more complicated,  and I can't see the gain. What do you see as the benifits?